CHANGELOG/OWNERS

```diff
@@ -9,14 +9,15 @@ approvers:
   - AuraSinis # 1.24 Release Notes Lead
   - cici37 # 1.23 Release Notes Lead
```

Comment:
@JamesLaverack @salaxander @palnabarun Do we need to cleanup any entry from this list or just adding the new one is fine ?

Reply:
For now, it's fine.